sanic/config.py

```diff
@@ -1,8 +1,16 @@
-import os
-import types
+from os import environ as os_environ
```

Comment:
Please, use `os.environ.get(...)` it's more clear what we're using, and in that case, we don't need an alias.

Reply:
But personally here I would prefer to import only environ from os - there is no need to import all os - it should be also a bit faster i suppose ?

Reply:
Will use
from os import environ